edXRegressionAutomated/Tests/LMS/test_loginpage.py

```diff
@@ -0,0 +1,22 @@
+from bok_choy.web_app_test import WebAppTest
+
+from pages.lms_login import LMSLoginPage
+from pages.dashboard import Dashboard
+from ..Settings.config import GlobalVariables
+
+
+class TestLMSLoginSuccess(WebAppTest):
+    def setUp(self):
+        super(TestLMSLoginSuccess, self).setUp()
+        self.browser.maximize_window()
```

Comment:
Maximizing window is not a great convention by default. It should only be used if you're focused on testing different screen resolutions.